modules/roles_profiles/manifests/profiles/disable_services.pp

```diff
@@ -3,64 +3,71 @@
-                    refreshonly => true;
-                'remove-index':
-                    command     => '/usr/bin/mdutil -a -E',
-                    refreshonly => true;
-            }
+      file { '/var/db/.spotlight-indexing-disabled':
+        content => 'indexing-disabled',
+        notify  => Exec['disable-indexing', 'remove-index'],
+      }
 
-            file { '/var/db/.spotlight-indexing-disabled':
-                content => 'indexing-disabled',
-                notify  => Exec['disable-indexing', 'remove-index'],
-            }
-
-            include macos_mobileconfig_profiles::disable_diagnostic_submissions
-            include macos_mobileconfig_profiles::disable_gatekeeper
+      include macos_mobileconfig_profiles::disable_diagnostic_submissions
+      include macos_mobileconfig_profiles::disable_gatekeeper
+    }
+    'Windows': {
+      include win_disable_services::disable_puppet
+      include win_disable_services::disable_windows_update
+      if $facts['os']['release']['full'] != '2012 R2' {
+        include win_disable_services::disable_wsearch
+        if ($facts['custom_win_location'] == 'azure') {
+          include win_scheduled_tasks::kill_local_clipboard
+        }
+        if $facts['custom_win_release_id'] == '2004' or '2009' {
+          include win_disable_services::disable_windows_defender_schtask
+        } else {
+          include win_disable_services::disable_windows_defender
         }
-        'Windows': {
-            include win_disable_services::disable_puppet
-            include win_disable_services::disable_windows_update
-            if $facts['os']['release']['full'] != '2012 R2' {
-                include win_disable_services::disable_wsearch
-                if ($facts['custom_win_location'] == 'azure') {
-                    include win_scheduled_tasks::kill_local_clipboard
-                }
-              if $facts['custom_win_release_id'] == '2004' or '2009'{
-                  include win_disable_services::disable_windows_defender_schtask
-              } else {
-                  include win_disable_services::disable_windows_defender
-              }
-            }
-            if $facts['os']['release']['full'] == '10' {
-                include win_disable_services::disable_onedrive
-            }
-            # May be needed for non-hardaware
-            # Commented out because this will break the auto restore
-            # include win_disable_services::disable_vss
-            # include win_disable_services::disable_system_restore
+      }
+      if $facts['os']['release']['full'] == '10' {
+        include win_disable_services::disable_onedrive
+      }
+      if ($facts['custom_win_location'] == 'datacenter') {
+        if ($facts['custom_win_purpose'] == 'tester') {
+          include win_disable_services::disable_onedrive
         }
-        default: {
-            fail("${::operatingsystem} not supported")
+      }
+      if ($facts['custom_win_location'] == 'home') {
```

Comment:
Should the conditional include `datacenter`?

Reply:
This was me testing at home, I'll remove this part.